### PR TITLE
document for HTMLBaseElement.target

### PR DESCRIPTION
**Pull request by ShubhamOulkar**
This PR adds document for HTMLBaseElement.target property. 

Part of the project to document all interoperable features ([mdn/discussions#476](https://github.com/orgs/mdn/discussions/476))

### CODE REVIEW

File: files/en-us/web/api/htmlbaseelement/target/index.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+browser-compat: api.HTMLBaseElement.target
+---
+
+{{ApiRef("HTML DOM")}}
+
+The `target` property of the {{domxref("HTMLBaseElement")}} interface is a string that represents default target tab to show resulting output for hyperlinks and form elements. It is a value of [`target`](/en-US/docs/Web/HTML/Element/base#target) attribute of the {{HTMLElement("base")}} element.
+
+## Value
+
+A string that represents target values `_blank`, `_self` (default), `_parent`, `_top`, `framename`.
```
Comment (teoli2003) [inline files/en-us/web/api/htmlbaseelement/target/index.md:15]:
```suggestion
A string representing the target. Its value can be:
- the name of a {{HTMLElement("frame")}}, 
- One of the [keyword with specific values](/en-US/docs/Web/HTML/Element/base#target):`_blank`, `_self`, `_parent`, or `_top`.
```

File: files/en-us/web/api/htmlbaseelement/target/index.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+title: "HTMLBaseElement: target property"
+short-title: target
+slug: Web/API/HTMLBaseElement/target
+page-type: web-api-instance-property
+browser-compat: api.HTMLBaseElement.target
+---
+
+{{ApiRef("HTML DOM")}}
+
+The `target` property of the {{domxref("HTMLBaseElement")}} interface is a string that represents default target tab to show resulting output for hyperlinks and form elements. It is a value of [`target`](/en-US/docs/Web/HTML/Element/base#target) attribute of the {{HTMLElement("base")}} element.
```
Comment (teoli2003) [inline files/en-us/web/api/htmlbaseelement/target/index.md:11]:
```suggestion
The `target` property of the {{domxref("HTMLBaseElement")}} interface is a string that represents the default target tab to show the resulting output for hyperlinks and form elements.

It reflects the [`target`](/en-US/docs/Web/HTML/Element/base#target) attribute of the {{HTMLElement("base")}} element.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
It took me some time to figure out how to describe the keyword without duplicating the text everywhere.

I think I found a way: listing the keyword and having a link to their detailed description.

(Side note that you often forget articles in your sentences)

File: files/en-us/web/api/htmlbaseelement/target/index.md (hunk 1 above)
Comment (ShubhamOulkar) [reply files/en-us/web/api/htmlbaseelement/target/index.md:15]: thank you **@teoli2003**

**Comment by ShubhamOulkar** [comment]
> (Side note that you often forget articles in your sentences)

Thank You @teoli2003 🙌

File: files/en-us/web/api/htmlbaseelement/target/index.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+The `target` property of the {{domxref("HTMLBaseElement")}} interface is a string that represents the default target tab to show the resulting output for hyperlinks and form elements.
+
+It reflects the [`target`](/en-US/docs/Web/HTML/Element/base#target) attribute of the {{HTMLElement("base")}} element.
+
+## Value
+
+A string representing the target. Its value can be:
+
+- The name of a {{HTMLElement("frame")}}
```
Comment (teoli2003) [inline files/en-us/web/api/htmlbaseelement/target/index.md:19]:
```suggestion
- The name of a {{HTMLElement("frame")}}.
```

**Review by teoli2003 — APPROVED** [review]
👍  Thank you

**Pull request merged**